linkinator.config.json

```diff
@@ -2,6 +2,7 @@
   "recurse": true,
   "skip": [
     "https://codecov.io/gh/googleapis/",
-    "www.googleapis.com"
+    "www.googleapis.com",
+    "https://cloud.google.com/monitoring/workspaces/tiers"
```

Comment:
That doesn't seem right.  What are we doing that's referencing that url?

Reply:
there's an internal tracking issue, this is a URL in the protos that ends up in the JSDoc strings ultimately (see: https://github.com/googleapis/nodejs-monitoring/blob/a2dbb0b8df925317f01a193038ad520f0c8c43e7/protos/google/monitoring/v3/common.proto#L423).